astropy/io/votable/tests/converter_test.py

```diff
@@ -41,9 +39,8 @@ def test_oversize_char():
 
 def test_char_mask():
     config = {'verify': 'exception'}
-    field = tree.Field(
-        None, name='c', datatype='char',
-        config=config)
+    field = tree.Field(None, name='c', arraysize='1', datatype='char',
```

Comment:
So, instead of catching the warning, I fix the call so it does not emit warning in the first place. Is this more desirable, @astrofrog ?